build-system/server.js

```diff
@@ -218,6 +220,62 @@ function liveListTombstone(liveList) {
   }
 }
 
+function getLiveBlogItem() {
+  var now = Date.now();
+  // Value is inclusive of both min and max values.
+  var range = (min, max) => {
+    var values = Array.apply(null, Array(max - min + 1)).map((_, i) => min + i);
+    return values[Math.round(Math.random() * (max - min))]
+  };
+  // Generate a 3 to 7 worded headline
+  var headline = bacon(range(3, 7));
```

Comment:
How about adding a http://placekitten.com/300/350 to some of the entries at random?